libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx

```diff
@@ -95,11 +143,52 @@ export const PaymentPlan = ({ application, field }: FieldBaseProps) => {
       <div>Eitthvað fór úrskeiðis, núverandi greiðsludreifing fannst ekki</div>
     )
 
+  const SliderDescriptor = () => {
+    if (!distributionData || distributionData?.payments?.length < 1) return null
+
+    const monthlyPayments = distributionData.payments[0].payment
+    const lastMonthsPayment =
+      distributionData.payments[distributionData.payments.length - 1].payment
+
+    if (monthlyPayments === lastMonthsPayment) {
+      return (
+        <Box display="flex" justifyContent="flexEnd">
+          <Text variant="small" fontWeight="semiBold">
+            <span>Greiðsla </span>
```

Comment:
Should be a translation message for now unless we start getting strings from the service, we will then need to readdress this issue.